runtime/src/main/java/io/quarkiverse/authzed/client/AuthzedContext.java

```diff
@@ -32,7 +34,9 @@ public static AuthzedContext create(AuthzedConfig config) {
     }
 
     private static ManagedChannel createChannel(AuthzedConfig config) {
-        NettyChannelBuilder builder = NettyChannelBuilder.forAddress(config.url.getHost(), config.url.getPort());
+        NettyChannelBuilder builder = NettyChannelBuilder.forAddress(config.url.getHost(), config.url.getPort())
+                .directExecutor().offloadExecutor(Infrastructure.getDefaultExecutor())
```

Comment:
Not sure you need the executor and offload executor. Did you try without?

Reply:
Yes, you are correct. The system uses grpc-nio-worker for the call with or without the settings. 

We dont need to set the directExecutor and offloadExecutor. I have pushed the code to remove them.